ztp/extra-manifests-builder/99-sync-time-once/build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
@shajmakh  are all the ztp files related to this PR ?

Reply:
the PR needed a rebase